src/libs/getEmailKeyboardType/index.js

```diff
@@ -0,0 +1,8 @@
+
+/**
+ * * Return the defualt keyboard type when running on Web/Mobile-Web/Desktop
+ * @return {String | null}
```

Comment:
There is no need to add the `| null`; we are 100% sure about the return type. Also, Web covers Mobile-Web so that we can omit that part. And, there's a little typo in the `default` word.

```suggestion
 * Return the default keyboard type when running on Web/Desktop
 * @return {String}
```

Reply:
@HorusGoul  - I have committed the code review changes.